ansible/roles/schulcloud-server-core/tasks/main.yml

```diff
@@ -37,6 +37,13 @@
       template: onepassword.yml.j2
     when: ONEPASSWORD_OPERATOR is defined and ONEPASSWORD_OPERATOR|bool
 
+  - name: Admin Api ingress
+    kubernetes.core.k8s:
+      kubeconfig: ~/.kube/config
+      namespace: "{{ NAMESPACE }}"
+      template: admin-api-ingress.yml.j2
+      apply: yes
```

Comment:
when: WITH_API_ADMIN is defined and WITH_API_ADMIN|bool
is missing, probably missing at the configmap and secret as well, but for the others it is present